Nos/Service/UNSAPI.swift

```diff
@@ -7,23 +7,29 @@
 
 import Foundation
 import Logger
+import Dependencies
 
 typealias JSONObject = [String: Any]
 typealias UNSNameID = String
 
 struct UNSNameRecord: Identifiable, Equatable {
     var name: UNSName
     var id: UNSNameID
+    var nostrPubKey: HexadecimalString?
 }
 
 enum UNSError: Error {
     case generic
     case noAccessToken
     case requiresPayment(URL)
     case nameTaken
+    case notAuthenticated
     case developer
+    case badResponse
 }
 
+// swiftlint:disable type_body_length
```

Comment:
Maybe we can disable this rule in the entire project (not in this PR necessarily). It seems we keep just disabling this rule each time it complains.

Reply:
It's true. Next time it pops up for either of us let's make that change.